third_party/skia/src/core/SkShaderCodeDictionary.h

```diff
@@ -10,6 +10,7 @@
 
 #include <array>
 #include <unordered_map>
+#include <string>
```

Comment:
Why is this added?

Reply:
Because the file uses `std::string` and doesn't have an include for it. This single line would be an upstreamable patch, assuming upstream trunk is still similar.